controllers/providers/aws/utils.go

```diff
@@ -116,3 +115,17 @@ func GetTemplateLatestVersion(templates []*ec2.LaunchTemplate, templateName stri
 	}
 	return "0"
 }
+
+func TagEC2instance(instanceID, tagKey, tagValue string, client ec2iface.EC2API) error {
```

Comment:
Can you make this a method of `AmazonClientSet` and place it under ec2.go?
Make sure you use the object instead of taking in a client

Reply:
Yup. I shall do that.